fhir-install/README-DB2.md

```diff
@@ -0,0 +1,120 @@
+# IBM FHIR Server Install - Using docker container db2 as data store
+
+Running fhir server integration tests using docker db2.
+
+## Prerequisites
+
+- [Docker]
+
+## Build
+- Access the fhir-install/docker directory;
+- Copy Dockerfile-fhirdb2 and create-database.sh to a newly created directory;
+- Rename Dockerfile-fhirdb2 to Dockerfile;
+- Using Docker Terminal, access the new directory and run:
+
+```sh
+docker build -t fhirserverdb2 . --squash
+```
+## Run
+ 
+Once the image is built, start it with:
+
+```sh
+docker run -itd --name fhirdb2 --privileged=true -p 50000:50000 -e LICENSE=accept -e DB2INST1_PASSWORD=change-password -v /db2 -rm fhirserverdb2
+```
+if docker fails to find the image by name, then please use the image id instead of name for this above command.
+
+## Configuration
+
+The db2 docker container is started with the default fhirdb database created, for the integration test, we need to create three more database as following:
+
+```sh
+docker exec -ti fhirdb2 bash -c "su - db2inst1"
+db2 CREATE DB study using codeset UTF-8 territory us PAGESIZE 32768
+db2 CREATE DB ref using codeset UTF-8 territory us PAGESIZE 32768
+db2 CREATE DB profile using codeset UTF-8 territory us PAGESIZE 32768
+```
```

Comment:
add this as a conditional value in the create-databases.sh?

Reply:
I thought about this also, but because some users may not really want to configure tenant1 or configure multi-datastore in their env. so I didn't add them to the create-database.sh